enhancements/installer/aws-lb-subnet-selection.md

```diff
@@ -0,0 +1,974 @@
+---
+title: aws-lb-subnet-selection
+authors:
+- "@gcs278"
+reviewers:
+- "@candita"
+- "@frobware"
+- "@rfredette"
+- "@alebedev87"
+- "@miheer"
+- "@Miciah"
+- "@mtulio"
+approvers:
+- "@patrickdillon"
+api-approvers:
+- None
+creation-date: 2024-05-29
+last-updated: 2024-09-04
+tracking-link:
+  - https://issues.redhat.com/browse/CORS-3440
+see-also:
+  - "enhancements/ingress/lb-subnet-selection-aws.md"
+replaces:
+  - "enhancements/installer/aws-customer-provided-subnets.md"
+superseded-by:
+---
+
+# Installer LoadBalancer Subnet Selection for AWS
+
+## Summary
+
+This enhancement extends the OpenShift Installer's install-config, enabling cluster admins to
+configure subnets for AWS load balancers created for their IngressControllers at install time.
+This proposal allows the install-time configuration of subnets for the `default` IngressController
+as well as the default subnets for any new IngressController.
+
+This enhancement also deprecates the existing install-config field `platform.aws.subnets`
+in favor of a more flexible configuration field that handles specifying subnets for both
+IngressControllers and the cluster infrastructure. These new subnet roles, referred to as
+IngressController-role subnets, cluster-role subnets, or dual-role subnets (for those serving both roles)
+are further defined in [Defining Subnet Roles](#defining-subnets-roles).
+
+## Definitions and Terminology
+
+### The `default` IngressController
+
+This proposal will refer to the IngressController that gets created automatically during installation and handles
+the platform routes (console, auth, canary, etc.) as the `default` IngressController. Note that
+"default IngressController subnets" refers to the default subnets used by all IngressControllers, including
+the `default` IngressController. This is different from "`default` IngressController subnets", which specifically
```

Comment:
I appreciate your giving this comment for clarity. I would just add that grammatically this distinction is made with hyphens. default-IngressController subnets would be the subnets for the `default` IngressController, and default IngressController subnets (without the hyphen) would be the default subnets. The hyphen denotes which word the adjective "default" modifies.